cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-petset.yaml

```diff
@@ -64,9 +65,18 @@ spec:
           - name: grafana-persistent-storage
             mountPath: /var
       volumes:
-      - name: influxdb-persistent-storage
-        persistentVolumeClaim:
-          claimName: influxdb-claim
       - name: grafana-persistent-storage
         emptyDir: {}
-  serviceName: monitoring-influxdb
+  volumeClaimTemplates:
+  - metadata:
+      name: influxdb-ps
```

Comment:
fyi, this will give you a pv with a reall long name. Something like `influxdb-ps-monitoring-influxdb-grafana-v3-{index}`